### PR TITLE
Add a filter by prefecture to positives api

### PR DESCRIPTION
**Pull request by kiyohome**
I added a process to create JSON files for each prefecture and a process to filter by prefecture.

#12

### CODE REVIEW

**Comment by kiyohome** [comment]
> As always, you work fast!
> But You should use Flake8, a lint. Because it fails with CI / CD

Ok.

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 .now
-.idea
\ No newline at end of file
+.idea
+.venv
```
[the remark this answers is not in the document]
Comment (kiyohome) [reply .gitignore]: I was using Pipenv, but I had set PIPENV_VENV_IN_PROJECT=true, so .venv was created in the project. I removed .venv from .gitignroe.

**Comment by ryo-ma** [comment]
I have added **data** directory to .gitignore.

**Comment by kiyohome** [comment]
Uh, sorry, committing the data directory causes conflicts, doesn't it?
I have modified it so that it does not conflict.

**Comment by ryo-ma** [comment]
Great work!

**Pull request merged**